Linkta-org/core
# Adds Functionality for Popular Topics to Generate a New LinktaFlow

## Description



This pull request adds functionality to generate a new `LinktaFlow` from hard-coded `PopularTopics`.

- Moved checkbox state logic from `UserInputForm` to Zustand's `UserInputStore` so it can be utilized to enable and disable `PopularTopics` as well.
- Refactored `UserInputForm` to utilize Zustand for managing checkbox and loader state.

Fixes #357 

## Type of change




## How Can this be tested? Testing Plan to review this PR



  - `PopularTopics.tsx`
  - Run `npm run dev`
  - Navigate to the `generate` page.
  - Click on a popular topic.
  - The loader will display until the new LinktaFlow is generated or an error occurs.
  - Checking and unchecking the checkbox will enable and disable the popular topics and the generate button.


## Checklist:




## Additional Information:
